### PR TITLE
appium section updated

### PR DESCRIPTION
**Pull request by abdullahawara**

### CODE REVIEW

File: docs/tools/appium/continuous-integration/jenkins_gradle.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ These values would then be sent through the appropriate DesiredCapabilities obje
     capabilities.setCapability(TestObjectCapabilities.TESTOBJECT_DEVICE, deviceId);
 {% endhighlight %}
 
+For more information check out our [Appium setup guide.](https://help.testobject.com/docs/tools/appium/setups)
```
Comment (ragog) [inline docs/tools/appium/continuous-integration/jenkins_gradle.md]: remove the line, the link has to be in some other location (maybe further up, when you are hinting at the basic/watcher setup)

File: docs/tools/appium/continuous-integration/jenkins_gradle.md [hunk 2]
```diff
@@ -106,6 +108,8 @@ The capabilities in your setup method should look like this:
     capabilities.setCapability("testobject_test_report_id", resultWatcher.getTestReportId());
 {% endhighlight %}
 
+For more information check out our [Appium setup guide.](https://help.testobject.com/docs/tools/appium/setups)
```
Comment (ragog) [inline docs/tools/appium/continuous-integration/jenkins_gradle.md]: same as above (with suite setup)

File: docs/tools/appium/setups/java-junit/setup-java-junit-basic.md [hunk 3]
```diff
@@ -43,9 +42,10 @@ public class BasicTestSetup {
 
 Along with the mandatory capabilities we have specified, you can send over some optional ones to customize your test runs:
 
-* testobject_appium_version, if you want to run your tests against an Appium version other than the default one (1.3.5)
-* testobject_test_name, if you want to specify a name for your test that will be displayed in place of the default one
-* testobject_suite_name, if you want to apply a label to your tests so that they are easier to group / find later on
+* <strong>testobject_app_id</strong>, if you want to use an app version other than the default one.
```
Comment (ragog) [inline docs/tools/appium/setups/java-junit/setup-java-junit-basic.md]: "active", not default (just for clarity)

File: docs/tools/appium/continuous-integration/jenkins_gradle.md (hunk 1 above)
Comment (abdullahawara) [inline docs/tools/appium/continuous-integration/jenkins_gradle.md]: Done.

File: docs/tools/appium/setups/java-junit/setup-java-junit-basic.md (hunk 3 above)
Comment (abdullahawara) [inline docs/tools/appium/setups/java-junit/setup-java-junit-basic.md]: Updated.

**Comment by ragog** [comment]
👍

**Pull request merged**